realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -462,6 +477,15 @@ private static String getManagementRealmUrl(URL authUrl) {
         }
     }
 
+    /**
+     * FIXME Javadoc
+     *
+     * @return
+     */
+    public PermissionManager getPermissionManager() {
```

Comment:
Behavior of this method is a bit confusing.
This method returns cached instance if the returned object of previous call on the same thread was not closed.

Why do we need to provide `PermissionManager.close()` to users?